architecture/data_provider/001-sequence-store-packages-windows-info.puml

```diff
@@ -0,0 +1,49 @@
+@startuml
+participant  SysInfo         as dataprovider
+participant  Syscollector    as syscollector
+collections  RegistryWindow  as registry
```

Comment:
Propper name is Windows Registry
```suggestion
collections  Windows Registry as registry
```

Reply:
Hi @jnasselle 

The plantuml tool doesn't allow me to place the space character for names.

Regards
Hanes